integration_tests/samples/oauth/oauth_v2.py

```diff
@@ -95,17 +54,44 @@ def oauth_callback():
     # Retrieve the auth code and state from the request params
     if "code" in request.args:
         state = request.args["state"]
-        if state_service.consume(state):
+        if state_store.consume(state):
             code = request.args["code"]
             client = WebClient()  # no prepared token needed for this app
-            response = client.oauth_v2_access(
+            oauth_response = client.oauth_v2_access(
```

Comment:
If a developer uses Bolt for Python, the developer doesn't need to implement this on their own. https://github.com/slackapi/bolt-python/blob/v0.5.3a0/slack_bolt/oauth/oauth_flow.py#L255-L302